appservice/src/slotFunctionOperations.ts

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { FunctionEnvelope, FunctionEnvelopeCollection } from 'azure-arm-website/lib/models';
+import { ISubscriptionContext } from 'vscode-azureextensionui';
+import { requestUtils } from './utils/requestUtils';
+
+/**
+ * Temporary workaround because this isn't in azure sdk yet
```

Comment:
Issue links in Function repo?

Reply:
You mean the azure sdk repo? I don't see the point in filing an issue when we're on the deprecated version of the sdk

Reply:
I meant having issues in the Functions repo so we remember to change these.  Though I guess we'll have to change most calls when we change APIs.

Reply:
Ah - added a comment: https://github.com/microsoft/vscode-azurefunctions/issues/1083#issuecomment-539223138